### DISCUSSION TITLE
Is it possible to use a keyboard plugged into MMP via usb-c port?

### DISCUSSION THREAD

**Discussion by vprasanth** [OnionUI/Onion#1159] [Q&A] [answered]
I booted up the terminal app and plugged in a keyboard and hoped for the best :) 

I couldn't really find anything on the net about this being _unsupported_ so was just curious if its just something that needs to be developed or it's actually physically not possible with how the port is wired to the board (i.e. its only setup for charging). Would be great if someone could clarify. 

Thank you!

**Comment by Aemiii91** [comment, marked as answer]
USB-C is charging only.
